functions/public/styles/index.css

```diff
@@ -120,4 +169,17 @@ body{
     margin: 0 20px;
     font-family: 'Cabin', sans-serif;
     list-style-type: none;
+}
+@media only screen and (max-width: 500px) {
+    .left-cover{
+        right: 0;
+        width: 95%;
+    }
+    .leftstrip{
+        width: 5%;
+    }
+    #left-contain{
+        margin-top: 90px;
+        transition: all ease-in-out 400ms;
+    }
 }
```

Comment:
Missing a new line here. Each code file must have a new empty line at the end

Reply:
noted